src/java.base/share/classes/java/util/BitSet.java

```diff
@@ -1015,8 +1015,10 @@ public void andNot(BitSet set) {
     }
 
     /**
-     * Returns the hash code value for this bit set. The hash code depends
-     * only on which bits are set within this {@code BitSet}.
+     * {@return the hash code value for this bit set}
```

Comment:
I tried to find the (non-obvious) spec for {@return ...}
Googling led me to the now embarrassingly outdated
https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html
https://www.oracle.com/java/technologies/javase/javadoc-tool.html#javadocdocuments
before finding the "real" well-maintained doc
https://docs.oracle.com/en/java/javase/20/docs/specs/javadoc/doc-comment-spec.html
(and that would benefit from TOC and inline anchors)

Improving these docs might be the most valuable engineering work that javadoc team could do!